op-node/p2p/gossip_test.go

```diff
@@ -101,6 +102,59 @@ func TestVerifyBlockSignature(t *testing.T) {
 	})
 }
 
+func TestVerifyBlockSignatureWithRemoteSigner(t *testing.T) {
+	t.Skipf("Implement mocking of client rpc calls")
+
+	logger := testlog.Logger(t, log.LevelCrit)
+	cfg := &rollup.Config{
+		L2ChainID: big.NewInt(100),
+	}
+	peerId := peer.ID("foo")
+	secrets, err := e2eutils.DefaultMnemonicConfig.Secrets()
```

Comment:
It would be nice to not rely on e2e utils, as we're only generating a single temporary key, and not a full set of chain operator secret keys. Geth `crypto` package has functions for creating keys.

Reply:
Thank you!